src/test/test-acpi-fpdt.c

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "acpi-fpdt.h"
+#include "tests.h"
+
+TEST(acpi_get_boot_usec) {
+        int r;
+        r =acpi_get_boot_usec((long unsigned int *) pid_get_start_time, (((long unsigned int *) pid_get_start_time) + 3));
```

Comment:
Please introduce local variables to store results, rather than using global variables.

Also, insert an empty line after variables declaration and function call.

Missing space after `=`.